.docker/images/nginx/tests/bootstrap.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * PHPUnit Bootstrap file.
+ */
+
+include_once __DIR__ . '/vendor/autoload.php';
+
+function get_curl_headers($path = "/", $opts = NULL)
+{
+  $uri = getenv('LOCALDEV_URL') ?: 'http://nginx:8080';
+
+  $response = null;
+
+  $path = '/' . ltrim($path, '/');
+
+  exec("docker-compose exec -T --env LAGOON_IMAGE_VERSION_PHP=7.1 test curl {$uri}{$path} -I {$opts} 2>/dev/null", $response);
+
+  if (empty($response)) {
+    return [];
+  }
+
+  $response = array_map('trim', $response);
+
+  foreach ($response as $line) {
+    if (strpos($line, 'HTTP') !== false) {
+      $part = explode(' ', $line);
+      $headers['Status'] = trim($part[1]);
+      continue;
+    }
+    $part = explode(':', $line);
+    if (count($part) == 2) {
+      $headers[$part[0]] = trim($part[1]);
+    }
+  }
+
+  return $headers;
+}
+
+function curl_get_content($path = "/", $opts = NULL)
+{
+  $uri = getenv('LOCALDEV_URL') ?: 'http://nginx:8080';
+
+  $response = null;
+  $path = '/' . ltrim($path, '/');
+
+  exec("docker-compose exec -T --env LAGOON_IMAGE_VERSION_PHP=7.1 test curl {$uri}{$path} {$opts} 2>/dev/null", $response);
```

Comment:
GovCMS8 is PHP7.2 by default

Reply:
Although.. actually what is this env var used for in this context? 

`LAGOON_IMAGE_VERSION_PHP` expects a value like `-v0.23.2`, (as per `.env.default`) - but I'm unclear why this is required?